test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/multi_line_test.go

```diff
@@ -69,8 +69,10 @@ func (s *MultiLineSuite) BeforeTest(suiteName, testName string) {
 	}, 1*time.Minute, 10*time.Second)
 
 	// Create a new log folder location
+	// s.Env().RemoteHost.MkdirAll(utils.LinuxLogsFolderPath)
```

Comment:
Is there a reason why we're not using these or that they're commented out?

Reply:
After testing, I've found that these cause the tests to fail as compared to manually creating the directory/files and setting the permissions with `chmod`.

Reply:
I'll remove them